### PR TITLE
Add points.

### PR DESCRIPTION
**Pull request by GithubPagesStuff**

### CODE REVIEW

**Comment by rbx-nsg** [comment]
You put this in the wrong directory

**Comment by GithubPagesStuff** [comment]
There.

**Comment by rbx-nsg** [comment]
There are many issues with this

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/*
+	FileName: LoadPlaceInfo.ashx.ts
+	Written By: Nikita Nikolaevich Petko
+	File Type: Module
+	Description: Load Place info script
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts:5]: You wrote this, I didn't, add your name, also change the FileName and description to suit this controller

File: .gitignore [hunk 2]
```diff
@@ -49,4 +49,5 @@ package-lock.json
 # We don't want the js'
 /Assemblies
 DeploymentTooling
-server.log
\ No newline at end of file
+server.log
+Lib/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/all-time.d.ts #fix a weird bug
```
Comment (rbx-nsg) [inline .gitignore:53]: Remove this

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+	***
+*/
+
+//import a from 'axios';
+var fs = require('fs')
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts:29]: DO NOT USE REQUIRE UNLESS THE RESPONSE IS VOLATILE!!
Use:
```ts
import filestream from 'fs';
```

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
+/*
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts:1]: This file shouldn't be called __PageIndex.ts, you can drop this and move to _P-userId.ts

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+	limitations under the License.
+
+	***
+*/
+
+//import a from 'axios';
+var fs = require('fs')
+// why points? i know it's super useless
+// but i can just guess what it does and implement it that way.
+import { _dirname } from '../../../../../../../../Roblox/Helpers/Constants/Directories';
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts:32]: Do not use this as it's implemented into Roblox.Api

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+import { SetPoints } from '../../../../../../../Helpers/WebHelpers/Points/Set';
+export default {
+	method: 'all',
+	func: async (_req, res) => {
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts:36]: import Request and Response from express-serve-static-core and make this function into:
```ts
func: (request: Request, response: Response) => { ... }
```

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts (hunk 4 above)
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts:1]: Implementation Notes:
This should use an ApiClient from 'http//points.api.sitetest4.robloxlabs.com/'

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+		if (_req.method === 'OPTIONS') return res.send();
+		if (_req.method !== 'POST') {
+			return res.status(502).send() // Make sure you're sending the right method
+		}
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts:40]: It should respond with a 405 when the method is incorrect

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+            } else {
+                e["allTimeScore"]=0
+            }
+            return res.status(200).send(JSON.stringify(e))
+            
+            
+        } catch(err) {
+         
+        }
+    },
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/__pageIndex.ts:72]: All of this "try catch" code is volatile, please fix, look at a different controller, such as an AbTesting controller for an example

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/all-time.ts [hunk 9]
```diff
@@ -25,23 +25,33 @@
 	***
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/all-time.ts:25]: Follow the same issues for the previous file

File: Roblox/Helpers/WebHelpers/Points/Get.ts [hunk 10]
```diff
@@ -0,0 +1,42 @@
+import fs from 'fs';
```
Comment (rbx-nsg) [inline Roblox/Helpers/WebHelpers/Points/Get.ts:1]: These helpers aren't reliable, and they don't take a namespace or class, fix them

File: Roblox/Helpers/WebHelpers/Points/Set.ts [hunk 11]
```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
```
Comment (rbx-nsg) [inline Roblox/Helpers/WebHelpers/Points/Set.ts:1]: These helpers aren't reliable, and they don't take a namespace or class, fix them

**Review by rbx-nsg — CHANGES_REQUESTED** [review]

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/_P-userId.ts [hunk 12]
```diff
@@ -1,8 +1,8 @@
 /*
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/_P-userId.ts:1]: Look at the way you have _P-userId/_P-userId here, move back change the path from Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/_P-userId.ts to Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId.ts

File: Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/_P-userId.ts [hunk 13]
```diff
@@ -26,10 +26,9 @@
 */
 
 //import a from 'axios';
-var fs = require('fs')
 // why points? i know it's super useless
 // but i can just guess what it does and implement it that way.
-import { _dirname } from '../../../../../../../../Roblox/Helpers/Constants/Directories';
+import { _dirname } from '../../../../../../../Helpers/Constants/Directories';
```
Comment (rbx-nsg) [inline Roblox/Controllers/Roblox.Points/v1/universes/_P-universeId/users/_P-userId/_P-userId.ts:31]: Wrong module, and a really cursed path

**Review by rbx-nsg — CHANGES_REQUESTED** [review]

**Comment by rbx-nsg** [comment]
Umm, you're going to have to put that onto a different pull request, because it will conflict with this source code

**Comment by GithubPagesStuff** [comment]
ok.

**Comment by rbx-nsg** [comment]
Wait for me to push the latest changes before making the new pull request so you can pull it into your repo

**Pull request closed without merging**